{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"reachabilityStartFailed"}],"languages":["swift"],"platforms":["iOS"]}]}],"schemaVersion":{"major":0,"minor":2,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/datacapturing\/synchronizer\/synchronizererror\/reachabilitystartfailed"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/org.cocoapods.DataCapturing\/documentation\/DataCapturing\/Synchronizer\/SynchronizerError\/reachabilityStartFailed","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"If starting reachability checks fails."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"reachabilityStartFailed"}],"title":"Synchronizer.SynchronizerError.reachabilityStartFailed","roleHeading":"Case","role":"symbol","symbolKind":"case","externalID":"s:13DataCapturing12SynchronizerC0C5ErrorO23reachabilityStartFailedyA2EmF","modules":[{"name":"DataCapturing"}]},"hierarchy":{"paths":[["doc:\/\/org.cocoapods.DataCapturing\/documentation\/DataCapturing","doc:\/\/org.cocoapods.DataCapturing\/documentation\/DataCapturing\/Synchronizer","doc:\/\/org.cocoapods.DataCapturing\/documentation\/DataCapturing\/Synchronizer\/SynchronizerError"]]},"references":{"doc://org.cocoapods.DataCapturing/documentation/DataCapturing/Synchronizer/SynchronizerError/reachabilityStartFailed":{"role":"symbol","title":"Synchronizer.SynchronizerError.reachabilityStartFailed","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"reachabilityStartFailed"}],"abstract":[{"type":"text","text":"If starting reachability checks fails."}],"identifier":"doc:\/\/org.cocoapods.DataCapturing\/documentation\/DataCapturing\/Synchronizer\/SynchronizerError\/reachabilityStartFailed","kind":"symbol","type":"topic","url":"\/documentation\/datacapturing\/synchronizer\/synchronizererror\/reachabilitystartfailed"},"doc://org.cocoapods.DataCapturing/documentation/DataCapturing/Synchronizer/SynchronizerError":{"role":"symbol","title":"Synchronizer.SynchronizerError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"SynchronizerError"}],"abstract":[{"type":"text","text":"Errors thrown during data synchronization."}],"identifier":"doc:\/\/org.cocoapods.DataCapturing\/documentation\/DataCapturing\/Synchronizer\/SynchronizerError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"SynchronizerError"}],"url":"\/documentation\/datacapturing\/synchronizer\/synchronizererror"},"doc://org.cocoapods.DataCapturing/documentation/DataCapturing/Synchronizer":{"role":"symbol","title":"Synchronizer","fragments":[{"kind":"keyword","text":"class"},{"kind":"text","text":" "},{"kind":"identifier","text":"Synchronizer"}],"abstract":[{"type":"text","text":"An instance of this class synchronizes captured measurements from persistent storage to a Cyface server."}],"identifier":"doc:\/\/org.cocoapods.DataCapturing\/documentation\/DataCapturing\/Synchronizer","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"Synchronizer"}],"url":"\/documentation\/datacapturing\/synchronizer"},"doc://org.cocoapods.DataCapturing/documentation/DataCapturing":{"role":"collection","title":"DataCapturing","abstract":[],"identifier":"doc:\/\/org.cocoapods.DataCapturing\/documentation\/DataCapturing","kind":"symbol","type":"topic","url":"\/documentation\/datacapturing"}}}